cmd/auth.go

```diff
@@ -244,15 +230,15 @@ func Cancel(username string, password string, yes bool) error {
 	err = auth.Delete(c, username)
 	cleanup := fmt.Errorf("\n%s %s\n\n", "409", "Conflict")
 	if reflect.DeepEqual(err, cleanup) {
```

Comment:
this could use some cleaning up. can you put an issue in the controller-sdk-go repo to have `auth.Delete` return a well-known error?